LingoClash/LingoClash/Model/DataManagers/DataManagerResources.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  DataManagerResources.swift
+//  LingoClash
+//
+//  Created by Sherwin Poh on 14/4/22.
+//
+
```

Comment:
created this because i needed to access the pkGames and queueEntries collection name in my PKGameMoveUpdater and PKGameMatchFinder. Decided to do it for the rest as well to be consistent